ui/tests/unit/models/capabilities-test.js

```diff
@@ -81,4 +81,18 @@ module('Unit | Model | capabilities', function(hooks) {
     assert.notOk(model.get('canDelete'));
     assert.notOk(model.get('canList'));
   });
+
+  test('it does not require sudo on sys/leases/revoke if update capability is present', function(assert) {
+    let model = run(() =>
+      this.owner.lookup('service:store').createRecord('capabilities', {
+        path: 'sys/leases/revoke',
+        capabilities: ['update', 'read'],
+      })
+    );
+    assert.ok(model.get('canRead'));
+    assert.notOk(model.get('canCreate'), 'sudo requires the capability to be set as well');
```

Comment:
I think this doesn't have to do with sudo, just that `create` wasn't in the capabilities list.

Reply:
it's in every test so I just left it